tests/Unit/Subscriber/Core/RegistratorSubscriberTest.php

```diff
@@ -104,6 +140,17 @@ public function testNodeFromRegistry()
         $this->subscriber->handleNodeFromRegistry($this->persistEvent->reveal());
     }
 
+    public function testReorderNodeFomRegistry()
```

Comment:
Actually I just wanted to give up on this, but now I have seen the following on another PR:
![image](https://cloud.githubusercontent.com/assets/405874/8818843/6fd0786c-3046-11e5-9cb5-9bf3f309d3c6.png)

This is exactly what happens if we force this policy, and this is also the reason why I have a very strong opinion against it.